modules/ROOT/pages/_partials/rn-known-issues-ref.adoc

```diff
@@ -3,15 +3,42 @@
 
 This section lists known issues with  {PRODUCT} {MINOR_VERSION_NO}:
 
-* In the _Managing and protecting APIs using API management_ Solution Pattern, users must have administrator privileges to create a product in 3scale. Administrator privileges are not required for other Solution Patterns.
+// https://issues.redhat.com/browse/INTLY-9939
+* In Managed Integration SSO, setting the email details for the admin user within the *Manage account* page fails.
++
+To work around this issue:
 
-* The `customer-admin` user cannot prevent any user with access to the {webapp} from changing the {walkthrough-name} subscription settings. Any user with access to the {webapp} can determine which {walkthrough-name}s are available in a cluster.
+. Log in to the {webapp}
+
+. Click the *Single Sign-On* link available under the *All Services* tab.
+
+. Click on *Administration Console*.
+
+. Select the master realm.
+
+. Select *users* from the sidebar.
+
+. Click *View all users*.
+
+. Edit the admin user.
+
+. Add the email details and click *Save*.
+
+// https://issues.redhat.com/browse/INTLY-7399
+* In {amq-online-name-short}, multiple messages are not distributed evenly among multiple consumers. This happens when:
+** Consumers are slow (taking 10-12 minutes to process each message)
+** Each slow consumer has a prefetch value of zero
 
 // This is https://issues.redhat.com/browse/INTLY-7031
 * The following Solution Pattern is not available in this release:
```

Comment:
Since this is only one, can we just say "The <name> Sol pat is not available"?

Reply:
Indeed we can. I've changed it to make it so.